go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/flashbots/mev-boost-relay
 
-go 1.21
+go 1.22
```

Comment:
If you want to get it merged in the flashbots repo, this might not work. You will need to downgrade your golang version.